tests/testClassifier.py

```diff
@@ -48,17 +51,13 @@ def test1(self):
         self.assertIn("063.000.000", self.records)
         entries, total, updates = classifier.classify(self.records["063.000.000"])
 
-        # Condor doesn't emit the the current year in records.  The classifier
-        # code assumes it's the current year, and prepends that, so we have
-        # to assume that here as well when testing for the date.
-        year = str(date.today().year)
         rec = entries[0]
         self.assertEqual(rec.condorId, "063.000.000")
         self.assertEqual(rec.dagNode, "A2")
-        self.assertEqual(rec.submitTime, year+"-10-17 19:59:57")
+        self.assertTrue(rec.submitTime.endswith("-10-17 19:59:57"))
```

Comment:
Not important, but I wonder if using `self.assertRegexpMatches()` would result in a clearer error message on failure (`assertTrue` will just report that the test failed but not give context).